superset-frontend/src/components/ListView/ListViewStyles.less

```diff
@@ -19,92 +19,127 @@
 
 @import '~stylesheets/less/variables.less';
 
-.superset-list-view {
-  .filter-dropdown {
-    margin-top: 20px;
-  }
+.superset-list-view-container {
+  text-align: center;
+
+  .superset-list-view {
+    text-align: left;
+    background-color: white;
+    border-radius: 4px 0;
+    overflow: scroll;
+    margin: 0 16px;
+    padding-bottom: 48px;
+
+    .filter-dropdown {
+      margin-top: 20px;
+    }
 
-  .filter-column {
-    height: 30px;
-    padding: 5px;
-    font-size: 16px;
-  }
+    .filter-column {
+      height: 30px;
+      padding: 5px;
+      font-size: 16px;
+    }
 
-  .filter-close {
-    height: 30px;
-    padding: 5px;
+    .filter-close {
+      height: 30px;
+      padding: 5px;
 
-    i {
-      font-size: 20px;
+      i {
+        font-size: 20px;
+      }
     }
-  }
 
-  .table-row-loader {
-    animation: shimmer 2s infinite;
-    background: linear-gradient(
-      to right,
-      #f6f7f8 0%,
-      #edeef1 20%,
-      #f6f7f8 40%,
-      #f6f7f8 100%
-    );
-    background-size: 1000px 100%;
-
-    span {
-      visibility: hidden;
+    .table-cell-loader {
+      position: relative;
+
+      .loading-bar {
+        background-color: #e7e7e7;
+        border-radius: 7px;
+
+        span {
+          visibility: hidden;
+        }
+      }
+
+      &:after {
+        position: absolute;
+        transform: translateY(-50%);
+        top: 50%;
+        left: 0;
+        content: '';
+        display: block;
+        width: 100%;
+        height: 48px;
+        background-image: linear-gradient(
+          100deg,
+          rgba(255, 255, 255, 0),
+          rgba(255, 255, 255, 0.5) 60%,
+          rgba(255, 255, 255, 0) 80%
```

Comment:
feels like we might have variables for these colors already, although i'm not sure. Maybe we should add them if not?

Reply:
we don't have them and I don't see a huge advantage to making them variables; they're just black and black at half opacity.

Reply:
FWIW, the LESS variables have pre-baked colors and opacity stops, so you can do things like:
`background-color: fade(@success, @opacity-heavy);`

Those opacity stops haven't been migrated to the JS Theme for Emotion, yet. Emotion doesn't have the fancy color mixins, so we'd have to find the nifty JS way to accomplish the same goal.